doc/admin-guide/plugins/prefetch.en.rst

```diff
@@ -162,7 +162,13 @@ the following URLs will be requested to be prefetched ::
 
 Note ``--fetch-path-pattern`` is a PCRE regex/capture pattern and
 ``{$2+2}`` is a mechanism to calculate the next path by adding or
-subtracting integer numbers.
+subtracting integer numbers.  The operands will be treated as unsigned
+32-bit integers.  Invalid numbers are treated as zeroes, and numbers
+too large will be interpreted as 2^32-1.  If subtraction results in
```

Comment:
Did you mean 64 not 32?

Also have you tried https://restructuredtext.documatt.com/element/superscript.html ?

Reply:
I chose 32-bit operands and 64-bit results here so that addition doesn't ever overflow.  That way there doesn't need to be a special rule for handling overflows when adding.

Thanks for pointing out the superscript element.  I've changed the exponent to a superscript.